client/src/components/LoginModal.js

```diff
@@ -13,7 +13,7 @@ import {
   Input,
 } from "@chakra-ui/react";
 
-const LoginModal = ({ loginModalOpen, openLoginModal, closeLoginModal, openSignupModal }) => {
+const LoginModal = ({ loginModalOpen, closeLoginModal, openSignupModal, setLoggedIn }) => {
```

Comment:
Hey @kristenking ; It seems openLoginModal does not exist within your changes. I'm thinking it should be added. Please let me know what your thoughts are. Thanks!

Reply:
@vmaineng openLoginModal is not used anywhere in our code and looks like it is redundant that is why it was removed. there is loginModalOpen carries the function to open our login model. Please, let me know if you have any diffrent information or seen where it is used.

Reply:
@kristenking it seems openLoginModal is being used in line 62 in your code as a function. Should it be deleted?